osf/models/node.py

```diff
@@ -656,7 +656,12 @@ def csl(self):  # formats node information into CSL format for citation parsing
             csl['DOI'] = doi
 
         if self.logs.exists():
-            csl['issued'] = datetime_to_csl(self.logs.latest().date)
+            citation_date = self.logs.latest().date
+
+            if self.is_preprint and self.preprints.get_queryset()[0].original_publication_date:
```

Comment:
Pretty sure I'm the reason `self.preprints.get_queryset()[0]` exists  -- You should be able to just do `self.preprints.first()` instead. 

Also take a look at [is_preprint](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L437) -- it is a bit misleading since it has some particular cases in which it is is false (e.g., the node is made private).

You might be better off doing:

```
if self.preprints.exists():
    citation_date = self.preprints.first().original_publication_date
```